crates/db_schema/src/impls/local_user.rs

```diff
@@ -49,18 +50,31 @@ impl LocalUser {
     pool: &mut DbPool<'_>,
   ) -> Result<Vec<Self>, Error> {
     let conn = &mut get_conn(pool).await?;
-    diesel::update(local_user)
-      .set(accepted_application.eq(true))
+    diesel::update(local_user::table)
+      .set(local_user::accepted_application.eq(true))
       .get_results::<Self>(conn)
       .await
   }
 
-  pub async fn is_email_taken(pool: &mut DbPool<'_>, email_: &str) -> Result<bool, Error> {
+  pub async fn delete_old_denied_local_users(pool: &mut DbPool<'_>) -> Result<usize, Error> {
+    let conn = &mut get_conn(pool).await?;
+
+    let old_denied_registrations = registration_application::table
+      .filter(registration_application::deny_reason.is_not_null())
+      .filter(registration_application::published.lt(now() - 1.week()))
```

Comment:
2 filters are :

- Older than a week ago
- Deny reason must exist

Reply:
Put this kind of thing as comment in the code.

Reply:
The deny reason is required to ensure that the application was rejected?

Reply:
Yep, I'll add some comments in a sec.